components/filter-components/MapFilters.vue

```diff
@@ -230,19 +245,10 @@
                   @filterOn="emitFilterOn($event)"
                 >
                 </normal-filters-list>
-
-                <button
-                  @click="selectAll(property, propertyItem)"
-                  class="inline-block rounded hover:bg-gray-300 text-black px-1 pb-1 pt-1 text-xs leading-normal border border-gray-200 hover:shadow-md hover:-translate-y-1 transform transition-all duration-200 focus:outline-none focus:ring focus:ring-blue-300 focus:ring-opacity-50"
-                >
-                  Select All
-                </button>
-                <button
-                  @click="clearAll(property, propertyItem)"
-                  class="inline-block rounded hover:bg-gray-300 text-black px-1 pb-1 pt-1 text-xs leading-normal border border-gray-200 hover:shadow-md hover:-translate-y-1 transform transition-all duration-200 focus:outline-none focus:ring focus:ring-blue-300 focus:ring-opacity-50"
-                >
-                  Clear All
-                </button>
+                <NuxtLink v-if="getGlossaryLabel(propertyItem.label)" :to="{ path: '/glossary', query: { category: getGlossaryLabel(propertyItem.label) } }" 
```

Comment:
This link has been added but, at least on the ritual objects page, the popover says we should go to the HCA site to get information on the ritual objects so this could be confusing.

What is the difference between the HCA glossary and ours? How can we give information to the reader to make an informed decision about which one to visit? Should we display both or just pick one?

Reply:
have moved all the descriptions into the json file from a few prs back

Reply:
ahh great, thanks Ruby. A good change and confirmed that the link has been updated to use our glossary. Cheers!